packages/unitables/src/bee/UnitablesBeeTable.tsx

```diff
@@ -54,6 +55,8 @@ export interface UnitablesBeeTable {
   onRowDuplicated: (args: { rowIndex: number }) => void;
   onRowReset: (args: { rowIndex: number }) => void;
   onRowDeleted: (args: { rowIndex: number }) => void;
+  rowsWidth: object;
```

Comment:
Not sure this is ideal...

Reply:
It's not. I forgot to change the type, the correct one for now is `ConfigInputRow`. It's not the ideal name, I'll change this too